lib/decimal.ex

```diff
@@ -1,3 +1,15 @@
+defmodule Decimal.Macros do
+  @moduledoc false
+
+  defmacro doc_since(version) do
```

Comment:
You should be able to do `defmacrop` if you wish to keep everything in the same module.

Reply:
we can't because it's basically:

```
defmodule Decimal do
  defmacrop doc_since(version) do
  end

  doc_since "1.5.0"
end
```